src/cdk/bidi/dir.ts

```diff
@@ -27,7 +28,7 @@ import {Direction, Directionality} from './directionality';
   exportAs: 'dir',
 })
 export class Dir implements Directionality {
-  /** Layout direction of the element. */
+  /** @docs-private */
   _dir: Direction = 'ltr';
```

Comment:
Shouldn't be needed since its prefixed with an underscore.

Reply:
Done